src/controller/model/device.ts

```diff
@@ -828,6 +829,23 @@ class Device extends Entity {
         }
         this._customClusters[name] = cluster;
     }
+
+    public async triggerBroadcast(endpoint: number, sourceEndpoint: number, destination: BroadcastAddress, direction: Zcl.Direction,
```

Comment:
```suggestion
    public async broadcastCommand(endpoint: number, sourceEndpoint: number, destination: BroadcastAddress, direction: Zcl.Direction,
```

To align with the naming in `endpoint` (e.g. `endpoint.command`)